sup3r/models/wind.py

```diff
@@ -12,7 +12,7 @@
 logger = logging.getLogger(__name__)
 
 
-class WindGan(Sup3rGan, AbstractWindInterface):
+class WindGan(AbstractWindInterface, Sup3rGan):
```

Comment:
oh nice! just reordering solves the inheritance priority? Great!!

Reply:
Yeah! Python's solution to the diamond problem!